src/pages/terraform/downloads/index.tsx

```diff
@@ -30,33 +34,34 @@ const TerraformDownloadsPage = (props: GeneratedProps): ReactElement => {
 }
 
 /**
- * Pulled from terraform-website/pages/downloads/index.jsx on 3/9/2022:
+ * Pulled from terraform-website/pages/downloads/index.jsx on 2022-03-09:
  * https://github.com/hashicorp/terraform-website/blob/master/pages/downloads/index.jsx#L55-L98
+ *
+ * Modified 2022-03-28 to replace semverGte with semverSatisfies,
+ * as the former seemed to return pre-releases, which is not what we want.
  */
-function filterOldVersions(props) {
-  if (!props?.props?.releases?.versions) {
-    return props
-  }
-
-  const versions = props.props.releases.versions
-
-  // versions is in the form of { [version]: { ...metadata } }
+function filterVersions(
+  versions: ReleasesAPIResponse['versions'],
+  versionRange: string
+): ReleasesAPIResponse['versions'] {
   // Filter by arbitrary & reasonable version cutoff
-  const filteredVersions = Object.keys(versions).filter((version) => {
-    if (!semverGte(version, VERSION_DOWNLOAD_CUTOFF)) {
-      return false
+  const filteredVersions = Object.keys(versions).filter(
+    (versionNumber: string) => {
+      console.log({
+        versionNumber,
+        satisfies: semverSatisfies(versionNumber, versionRange),
+      })
+      return semverSatisfies(versionNumber, versionRange)
     }
-    return true
-  })
-
-  /** @type {{[x: string]:{ [y: string]: any}}} */
-  const tree = {}
+  )
 
   /**
    * Computes the latest patch versions for each major/minor
-   * e.g. given [1.1.2, 1.1.1, 1.1.0, 1.0.9, 1.0.8] -> return [1.1.2, 1.0.9]
+   * e.g. given [1.1.2, 1.1.1, 1.1.0, 1.0.9, 1.0.8]
+   * return [1.1.2, 1.0.9]
```

Comment:
Added some comments on this part since I had a bit of trouble parsing what was going on initially.